### DISCUSSION TITLE
Permet de remplissage de champs même si le formulaire est déjà débuté

### DISCUSSION THREAD

**Discussion by ChrisThe01** [MTESSDev/FRW#38] [Annonces]
Bonjour,

Dans le projet SGD, nous avons un besoin d'où il faut permettre de faire du préremplissage de champs dans le traitement FRW113 (par exemple, l'adresse, nom, prénom, etc.).

Bonne journée,

**Comment by billo04** [comment]
Bonjour,
Nous voudrions savoir si cette demande a été prise en charge.
Merci

**Comment by vinnyboii** [comment]
Bonjour,
Nous avons cette demande dans notre carnet effectivement.

Elle sera prise en charge dans les prochaines semaines.

Cependant, nous ne permettrons pas de modifier n'importe quel champ lors de la reprise pour des raisons d'intégrité d'information.

On vous reviendra avec la solution.

Merci

Vincent

**Comment by vinnyboii** [comment]
La fonctionnalité est disponible en IT.
@ChrisThe01 @billo04
